### ISSUE TITLE
Support MySQL/MariaDB?

### ISSUE THREAD

**Issue by halybang** [hexya-erp/hexya#6]
Do you have any plan to support MySQL/MariaDB?

**Comment by npiganeau** [comment]
Yes, all DB actions are implemented through an interface so that we can easily create a new adapter for any database that have a go driver compatible with `sql/driver`.

Maybe I hardcoded some stuff which are particular to Postgres, but they will be easily moved out.

However, this is not a high priority for the moment.

**Comment by songwutk** [comment]
ERP should run on more stability like PostgreSQL not recommend MySQL or MariaDB.I found that 200GB database size, postgreSQL query much faster.
